### PR TITLE
VB-1424, stacked book another visit button

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
## Description
Changed new 'Book another visit for this prisoner' button to stacked display

### CODE REVIEW

File: server/views/pages/bookAVisit/confirmation.njk [hunk 1]
```diff
@@ -50,11 +50,12 @@
         {% endif %}
 
         {{ govukButton({
-            classes: "govuk-!-margin-top-3, govuk-!-margin-bottom-6",
+            classes: "govuk-!-margin-top-3, govuk-!-margin-bottom-4",
             text: "Go to manage prison visits",
             href: "/",
             preventDoubleClick: true
         }) }}
+        <br>
```
Comment (psoleckimoj) [inline server/views/pages/bookAVisit/confirmation.njk:58]: Being a pain for you but `<br>` is for line breaks in text and not CSS layout, ideally want wrapping in a `<div>` or similar to stop them floating next to each other 🙂

Comment (hutcheonb-moj) [reply server/views/pages/bookAVisit/confirmation.njk]: Fair point! Done :)

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**